### PR TITLE
Navjeevan kaur

### PR DESCRIPTION
**Pull request by navjeevan24**
Hey @dipanshu42 can you please request to @akay08923 to review my code. Thanks

### CODE REVIEW

File: library/returnBook/rETURN_bOOK_cONTROL.java [hunk 1]
```diff
@@ -4,84 +4,121 @@
...
-	private enum controlState { INITIALISED, READY, INSPECTING };
+	private enum ControlState { INITIALISED, READY, INSPECTING };
 	//private cOnTrOl_sTaTe sTaTe;
-	private controlState state;
+	private ControlState state;
 	
-	private Library lIbRaRy;
-	private library Library;
+	//private Library lIbRaRy;
+	private Library Iibrary;
```
Comment (akay08923) [inline library/returnBook/rETURN_bOOK_cONTROL.java:17]: there is spelling mistake. Iibrary should library.

File: library/returnBook/rETURN_bOOK_cONTROL.java [hunk 2]
```diff
@@ -4,84 +4,121 @@
...
-		sTaTe = cOnTrOl_sTaTe.INSPECTING;		
+		//if (CurrENT_loan.Is_OvEr_DuE()) 
+		if (currentLoan.isOverdue());
+			//Ui.DiSpLaY(String.format("\nOverdue fine : $%.2f", Over_Due_Fine));
+		         ui.display(String.format("\nOverdue fine : $%.2f", overdueFine));
+		
+		//Ui.sEt_sTaTe(ReturnBookUI.uI_sTaTe.INSPECTING);
+		ui.setState(returnBookUi.uiState.INSPECTING);
+		//sTaTe = cOnTrOl_sTaTe.INSPECTING;
+		state = controlState.INSPECTING;
```
Comment (akay08923) [inline library/returnBook/rETURN_bOOK_cONTROL.java:92]: controlState should be ControlState

File: library/returnBook/rETURN_bOOK_cONTROL.java [hunk 3]
```diff
@@ -4,84 +4,121 @@
...
+		ui.setState(ReturnBookUi.uiState.COMPLETED);
 	}
 
 
-	public void dIsChArGe_lOaN(boolean iS_dAmAgEd) {
-		if (!sTaTe.equals(cOnTrOl_sTaTe.INSPECTING)) 
+	//public void dIsChArGe_lOaN(boolean iS_dAmAgEd) {
+	public void dischargeLoan(boolean isDamaged) {
+		//if (!sTaTe.equals(cOnTrOl_sTaTe.INSPECTING))
+		if (!state.equals(controlState.INSPECTING)
```
Comment (akay08923) [inline library/returnBook/rETURN_bOOK_cONTROL.java:111]: controlState should be ControlState

File: library/returnBook/rETURN_bOOK_cONTROL.java [hunk 4]
```diff
@@ -4,84 +4,121 @@
...
-		Ui.sEt_sTaTe(ReturnBookUI.uI_sTaTe.READY);
-		sTaTe = cOnTrOl_sTaTe.READY;				
+		//lIbRaRy.DiScHaRgE_LoAn(CurrENT_loan, iS_dAmAgEd);
+		    libaray.discahrgeLoan(currentLoan, isDamaged);
+		//CurrENT_loan = null;
+	         currentLoan = null;
+		//Ui.sEt_sTaTe(ReturnBookUI.uI_sTaTe.READY);
+	         ui.setState(returnBookUi.uiState.READY);
+		//sTaTe = cOnTrOl_sTaTe.READY;	
+		state = controlState.READY;
```
Comment (akay08923) [inline library/returnBook/rETURN_bOOK_cONTROL.java:121]: controlState should be ControlState

**Review by akay08923 — COMMENTED** [review]
Hi @dipanshu42 , i have found some mistakes in the changed code. It just a class name. I request you to tell @navjeevan24 about these mistakes so that she can make it correct.

File: library/returnBook/rETURN_bOOK_cONTROL.java (hunk 1 above)
Comment (navjeevan24) [reply library/returnBook/rETURN_bOOK_cONTROL.java:17]: i changed it

File: library/returnBook/rETURN_bOOK_cONTROL.java (hunk 2 above)
Comment (navjeevan24) [reply library/returnBook/rETURN_bOOK_cONTROL.java:92]: done changes

File: library/returnBook/rETURN_bOOK_cONTROL.java (hunk 4 above)
Comment (navjeevan24) [reply library/returnBook/rETURN_bOOK_cONTROL.java:121]: done changes.

**Review by navjeevan24 — COMMENTED** [review]
hey @dipanshu42 I did changes according to your comments please tell to @akay08923 to review it. Thanks.

**Comment by dipanshu42** [comment]
Okay great i'll let ankush know

**Review by akay08923 — APPROVED** [review]

**Comment by akay08923** [comment]
Hi @dipanshu42 , i checked the code now Everything is correct. I approved the code. Now she (@navjeevan24)  can merge the code with master branch.

**Comment by dipanshu42** [comment]
Already did mate. No need to worry about it

**Comment by akay08923** [comment]
Hi Dipanshu, I have checked the code ReturnBookControl class. Now
everything is fine. Code is meeting style guidelines. So she can merged the
code with master branch.
Thanks

**Pull request merged**